### DISCUSSION TITLE
Facebook test app

### DISCUSSION THREAD

**Discussion by glauberssoutodossantos** [chatwoot/chatwoot#4872] [Self hosted]
Hello,

I'm using a facebook testing app to do app approval. For this I cloned my original app, as per the facebook reviewer's orientation, as shown below.

![image](https://user-images.githubusercontent.com/99494469/173832521-0148dd52-815e-4941-bd6e-c1a7b28357c9.png)

This clone app has an FB_APP_SECRET and FB_APP_ID. So far without any problems, I parameterized correctly in my application and deployed it.

But when I'm trying to test it's showing a blank page in facebook login, as below.

![image](https://user-images.githubusercontent.com/99494469/173833056-750eba1a-d223-40f0-b2e6-2684e1544676.png)

URL: https://web.facebook.com/v13.0/dialog/oauth?app_id=1415057262326722&cbt=1655298036889&channel_url=https%3A%2F%2Fstaticxx.facebook.com%2Fx%2Fconnect%2Fxd_arbiter%2F%3Fversion%3D46%23cb%3Df38f9fc0fd4d878%26domain%3Dapp.leadinbox.com.br%26is_canvas%3Dfalse%26origin%3Dhttps%253A%252F%252Fapp.leadinbox.com.br%252Ff2f2bed7c7ffe9c%26relation%3Dopener&client_id=1415057262326722&display=popup&domain=app.leadinbox.com.br&e2e=%7B%7D&fallback_redirect_uri=https%3A%2F%2Fapp.leadinbox.com.br%2Fapp%2Faccounts%2F1%2Fsettings%2Finboxes%2Fnew%2Ffacebook%23&locale=en_US&logger_id=f29b18889313b4&origin=1&redirect_uri=https%3A%2F%2Fstaticxx.facebook.com%2Fx%2Fconnect%2Fxd_arbiter%2F%3Fversion%3D46%23cb%3Df10f6361d98b858%26domain%3Dapp.leadinbox.com.br%26is_canvas%3Dfalse%26origin%3Dhttps%253A%252F%252Fapp.leadinbox.com.br%252Ff2f2bed7c7ffe9c%26relation%3Dopener%26frame%3Df1aa2ee1f5e4d3c&response_type=token%2Csigned_request%2Cgraph_domain&scope=pages_manage_metadata%2Cpages_messaging%2Cinstagram_basic%2Cpages_show_list%2Cpages_read_engagement%2Cinstagram_manage_messages&sdk=joey&version=v13.0&ret=login&fbapp_pres=0&tp=unspecified&ext=1655301641&hash=AeZsVNsXcmXMWYydEEc

Any tips on how to submit the app for facebook approval as a test?

**Comment by grupale** [comment]
Hello, I think there is a pending update to support Facebook api v14

**Reply by glauberssoutodossantos** [reply]
Its alredy in v14

![image](https://user-images.githubusercontent.com/99494469/174338582-59aaf486-518f-4ee1-ad81-5c984740e8b1.png)

**Reply by sojan-official** [reply]
@glauberssoutodossantos could you update chatwoot to v2.6.0

**Comment by qqenglishbr** [comment]
I'm having the same problem, a grey screen when I tried to create a messenger inbox.
API v.14 and chatwoot v2.11.0
